composer.json

```diff
@@ -26,7 +26,7 @@
         "doctrine/migrations": "^3.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.0|^8.0|^9.0",
+        "phpunit/phpunit": "^8.0|^9.0",
```

Comment:
We could also use `^8.5.21|^9.5` here. We don't gain much from running the `lowest` job with PHPUnit 8.0.0.